code/modules/mob/living/silicon/robot/robot_items.dm

```diff
@@ -139,54 +139,14 @@
 		user << "Harvesting \a [target] is not the purpose of this tool.  The [src] is for plants being grown."
 
 // A special tray for the service droid. Allow droid to pick up and drop items as if they were using the tray normally
-// Click on table to unload, click on item to load. Otherwise works identically to a tray.
-// Unlike the base item "tray", robotrays ONLY pick up food, drinks and condiments.
+// Click on table to unload, click on item to load. Alt+click to load everything on tile
 
 /obj/item/weapon/tray/robotray
 	name = "RoboTray"
 	desc = "An autoloading tray specialized for carrying refreshments."
 
 /obj/item/weapon/tray/robotray/afterattack(atom/target, mob/user as mob, proximity)
-	if(!proximity)
```

Comment:
A lot of deletions because most of the stuff in this file was centralised in the base tray, and now it mostly just calls base procs instead of snowflake code. robo trays work perfectly